packages/next/client/future/image.tsx

```diff
@@ -520,7 +531,10 @@ export default function Image({
   }
   const configContext = useContext(ImageConfigContext)
   const config: ImageConfig = useMemo(() => {
-    const c = configEnv || configContext || imageConfigDefault
+    const c = Object.assign(
+      configEnv || configContext || imageConfigDefault,
+      globalImageConfig
+    )
     const allSizes = [...c.deviceSizes, ...c.imageSizes].sort((a, b) => a - b)
     const deviceSizes = c.deviceSizes.sort((a, b) => a - b)
     return { ...c, allSizes, deviceSizes }
```

Comment:
You end up creating a new object here so you probably shouldn't mutate the base config objects. they are irreversible (meaning if you unset the global config the changes won't ever be reset to defaults)